packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -174,29 +225,27 @@ export class PersonalizeMiddleware extends MiddlewareBase {
       return response;
     }
 
-    if (!browserId) {
-      browserId = await this.cdpService.generateBrowserId();
-
-      if (!browserId) {
-        debug.personalize('skipped (browser id generation failed)');
-        return response;
-      }
-    }
-
-    // Execute targeted experience in CDP
-    const { ua } = userAgent(req);
     const params = this.getExperienceParams(req);
-    const pointOfSale = this.config.getPointOfSale
-      ? this.config.getPointOfSale(site, language)
-      : PosResolver.resolve(site, language);
-    const variantId = await this.cdpService.executeExperience(
+    const { ua } = userAgent(req);
+
+    debug.personalize(
+      'executing experience for %s %s %s %o',
```

Comment:
Check the debug log parameters, looks like we have 1 too many string `%s` params?